Reyes_PaulAngelo/works/script.js

```diff
@@ -0,0 +1,157 @@
+
+  submitCommentButton.addEventListener("click", function (e) {
+    e.preventDefault();
+    const name = commentNameInput.value.trim();
+    const text = commentTextInput.value.trim();
+
+    if (!name || !text) {
+      return;
+    }
+
+    addComment(name, text, new Date());
+    commentForm.reset();
+    checkCommentValidity();
+  });
+
+  function addComment(name, text, date) {
+    const commentElement = document.createElement("p");
+    commentElement.innerHTML =
+      "Name: " +
+      name +
+      "<p>Comment: " +
+      text +
+      "<p><small>" +
+      date.toLocaleString() +
+      "</small>";
+    commentsContainer.appendChild(commentElement);
+
+    comments.push({ name: name, text: text, date: date });
+  }
+
+  function sortComments(ascending) {
+    comments.sort(function (a, b) {
+      return ascending ? a.date - b.date : b.date - a.date;
+    });
+    renderComments();
+  }
+
+  function renderComments() {
+    commentsContainer.innerHTML = "";
+    comments.forEach(function (comment) {
+      const commentElement = document.createElement("p");
+      commentElement.innerHTML =
+        "Name: " +
+        comment.name +
+        "<p>Comment: " +
+        comment.text +
+        "<p><small>" +
+        comment.date.toLocaleString() +
+        "</small>";
+      commentsContainer.appendChild(commentElement);
+    });
+  }
+
+  sortAscButton.addEventListener("click", function () {
+    sortComments(true);
+    sortAscButton.blur();
+  });
+
+  sortDescButton.addEventListener("click", function () {
+    sortComments(false);
+    sortDescButton.blur();
+  });
+});
+
+function searchCountry() {
+  let countryName = document.getElementById("countryInput").value.trim();
+  if (!countryName) {
+    document.getElementById("countryDetails").innerHTML =
+      "<p>Please enter a country name.</p>";
+    document.getElementById("sameRegionCountries").innerHTML = "";
+    return;
+  }
+
+  fetch("https://restcountries.com/v3.1/name/" + countryName)
+    .then(function (response) {
+      if (!response.ok) {
+        throw new Error("Country not found");
+      }
+      return response.json();
+    })
+    .then(function (countryData) {
+      let country = countryData[0];
+      let details = `
+        <h2>Country Details - ${country.name.common}</h2>
+        <p><strong>Capital:</strong> 
+        ${country.capital ? country.capital[0] : "N/A"}</p>
+        <p><strong>Population:</strong> 
+        ${country.population ? country.population.toLocaleString() : "N/A"}</p>
+        <p><strong>Region:</strong> 
+        ${country.region ? country.region : "N/A"}</p>
+        <p><strong>Languages:</strong> 
+        ${
+          country.languages
+            ? Object.values(country.languages).join(", ")
+            : "N/A"
+        }</p>
+        <p><strong>Timezones:</strong> 
+        ${country.timezones ? country.timezones.join(", ") : "N/A"}</p>
+      `;
+      document.getElementById("countryDetails").innerHTML = details;
```

Comment:
IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5